MAUI/DataGrid/Clipboard-Operations.md

```diff
@@ -0,0 +1,546 @@
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.PasteCellContent += dataGrid_PasteCellContent;
+
+void dataGrid_DataGridPasteCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+The below code example change the clipboard value as Test instead of clipboard value BOLID.
+
+{% tabs %}
+{% highlight c# %}
+void dataGrid_DataGridPasteCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+    if (e.Column.MappingName == "Customer" && (e.RowData as OrderInfo).Customer == "Welli")
+        e.ClipBoardValue = "Test";
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Paste Clipboard Content based on Cell Value in Maui DataGrid" src="Images/clipboard-operations/maui-datagrid-paste-cell-value.png" width="404"/>
+
+The below code example handled the paste operation when `MappingName` of Column is OrderID
+
+{% tabs %}
+{% highlight c# %}
+void dataGrid_DataGridPasteCellContent(object sender, DataGridCopyPasteCellEventArgs e)
+{
+    if (e.Column.MappingName == "OrderID")
+        e.Handled = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Paste Clipboard Content based on Mapping Name in Maui DataGrid" src="Images/clipboard-operations/maui-datagrid-paste-mapping-name.png" width="404"/>
+
+## Handling Programmatically
+
+### Programmatically Copy to Clipboard in Maui DataGrid
+
+Copy the selected records/cells in SfDataGrid by using Copy method in CopyPasteController of SfDataGrid.
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.CopyPasteController.Copy();
+{% endhighlight %}
+{% endtabs %}
+
+Copy a record by selecting the record using MoveCurrentCellTo method and `Copy` method in CopyPasteController of SfDataGrid.
+
+{% tabs %}
+{% highlight c# %}
+RowColumnIndex rowColumnIndex = new RowColumnIndex();
+var lastSelectedIndex = dataGrid.CurrentCellManager.RowColumnIndex.RowIndex;
+rowColumnIndex.RowIndex = 2;
+rowColumnIndex.ColumnIndex = 2;
+this.dataGrid.MoveCurrentCellTo(rowColumnIndex);
+this.dataGrid.CopyPasteController.Copy();
+{% endhighlight %}
+{% endtabs %}
+
+Copy the multiple cells by selecting group of cells using SelectCells method and `Copy` method in `CopyPasteController` of SfDataGrid.
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.SelectCells(this.dataGrid.GetRowGenerator().Items[2].RowData, this.dataGrid.Columns[1], this.dataGrid.GetRowGenerator().Items[5].RowData, this.dataGrid.Columns[3]);
+this.dataGrid.CopyPasteController.Copy();
+{% endhighlight %}
+{% endtabs %}
+
+### Copy rows without selecting in Maui DataGrid
+
+You can copy the records without selection by using CopyRowsToClipboard method in `CopyPasteController` of SfDataGrid.
+ 
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.CopyPasteController.CopyRowsToClipboard(2, 4);
+{% endhighlight %}
+{% endtabs %}
+
+### Programmatically Cut Data to Clipboard in Maui DataGrid
+
+Cut the selected records/cells in SfDataGrid by using Cut method in `CopyPasteController` of SfDataGrid.
```

Comment:
To cut the selected records/cells in SfDataGrid, use the Cut method in `CopyPasteController`: